examples/forms.amp.html

```diff
@@ -28,6 +26,21 @@
             display: block;
             color: #4CAF50;
         }
+
+        form.amp-form-submitting .submitting-message {
```

Comment:
I actually just was thinking to add `amp-form-*` classes and not define default styling yet. The reason that the class names immediately become controversial, but the `amp-form-*` classes are immediately usable.

Alternatively, we need to bikeshed all the "form-message" class names and probably add prefixes too. This is different from `ui-invalid` and such since we have an upcoming spec to bikeshed naming for us :)